ras_backstage/ras_config/ras_config.py

```diff
@@ -0,0 +1,123 @@
+import json
+from os import getenv
+
+import yaml
+from structlog import get_logger
+
+log = get_logger()
+
+
```

Comment:
Is this module from common?

Reply:
yes, changed the logging method to be inline with the rest of this service.